src/types/apiInterfaces.ts

```diff
@@ -233,7 +245,6 @@ export interface IApiErrorResponse {
     field: string; // Todo - Check with PAPI the list of possible fields to declare const and types
     severity: string; // Todo - Check with PAPI the list of possible severities to declare const and types
     sub_type: string; // Todo - Check with PAPI the list of possible sub_types to declare const and types
-    address_type?: string;
```

Comment:
question: why `address_type` get removed here?

Reply:
Good catch. This is not intentional.